packages/server/src/api/app/bsky/updateProfile.ts

```diff
@@ -0,0 +1,118 @@
+import { Server } from '../../../lexicon'
+import { AuthRequiredError, InvalidRequestError } from '@adxp/xrpc-server'
+import * as locals from '../../../locals'
+import { AdxUri } from '@adxp/uri'
+
+const profileNsid = 'app.bsky.profile'
```

Comment:
Would love to be able to steal these out of lexicon.  Maybe we could generate an enum of nsids 🤔

Reply:
yup i'd definitely like that as well. would be nice at the db layer as well